bench/wtperf/wtperf.c

```diff
@@ -2091,6 +2335,18 @@ main(int argc, char *argv[])
 	if ((ret = config_compress(cfg)) != 0)
 		goto err;
 
+	/* You can't have truncate on a random collection */
+	if (cfg->has_truncate && cfg->random_range) {
+		fprintf(stderr, "Cannot run truncate and random_range\n");
```

Comment:
This should use the wtperf ```lprintf``` error logging function. Similarly to other occurrences of printf and fprintf in this diff.